src/Illuminate/Http/Client/Response.php

```diff
@@ -204,12 +204,27 @@ public function toPsrResponse()
      * Throw an exception if a server or client error occurred.
      *
      * @return $this
-     *
-     * @throws \Illuminate\Http\Client\RequestException
```

Comment:
this should be kept

Reply:
Thanks! Always forget if these should be applied to all calling places as well, or only the method throwing.